### DISCUSSION TITLE
How much weight can this rover carry?

### DISCUSSION THREAD

**Discussion by ivanperez-keera** [nasa-jpl/open-source-rover#452] [General]
How much weight can this rover carry atm?

**Comment by Achllle** [comment]
![image](https://github.com/nasa-jpl/open-source-rover/assets/14060847/827e2496-7dae-4f33-bdf8-eeed435ea506)
While I've tested it at higher payloads of 25kg, I'd say the current version shouldn't be loaded with more than 10kg for longer distances.
